marconi-chain-index/test/Spec/Marconi/ChainIndex/Indexers/Utxo/UtxoIndex.hs

```diff
@@ -3,82 +3,77 @@
-import Control.Lens.Operators ((^.))
-import Control.Monad (forM_, void)
+import Control.Lens.Operators ((^..))
+import Control.Monad (forM, forM_, void)
 import Control.Monad.IO.Class (liftIO)
 import Data.Aeson qualified as Aeson
 import Data.ByteString (ByteString)
+import Data.Functor ((<&>))
 import Data.List qualified as List
 import Data.List.NonEmpty (nonEmpty)
-import Data.Maybe (fromJust, isJust, isNothing, mapMaybe)
+import Data.Map (Map)
+import Data.Map qualified
+import Data.Maybe (fromJust, fromMaybe, isJust, isNothing, mapMaybe)
 import Data.Proxy (Proxy (Proxy))
 import Data.Set (Set)
 import Data.Set qualified as Set
-import Database.SQLite.Simple qualified as SQL
-import Gen.Marconi.ChainIndex.Indexers.Utxo (genEventWithShelleyAddressAtChainPoint, genUtxoEvents)
+
+import Cardano.Api qualified as C
+import Gen.Marconi.ChainIndex.Indexers.Utxo (genShelleyEraUtxoEvents, genUtxoEvents)
 import Gen.Marconi.ChainIndex.Indexers.Utxo qualified as UtxoGen
 import Gen.Marconi.ChainIndex.Mockchain (mockBlockTxs)
-import Hedgehog (Property, cover, forAll, property, (===))
-import Hedgehog qualified
-import Hedgehog.Gen qualified as Gen
-import Hedgehog.Range qualified as Range
 import Helpers (addressAnyToShelley)
 import Marconi.ChainIndex.Indexers.Utxo (StorableEvent (ueInputs, ueUtxos))
 import Marconi.ChainIndex.Indexers.Utxo qualified as Utxo
 import Marconi.ChainIndex.Types (TargetAddresses)
 import Marconi.Core.Storable (StorableQuery)
 import Marconi.Core.Storable qualified as Storable
+
+import Hedgehog (Gen, Property, cover, forAll, property, (===))
+import Hedgehog qualified
+import Hedgehog.Gen qualified as Gen
+import Hedgehog.Range qualified as Range
 import Test.Tasty (TestTree, testGroup)
 import Test.Tasty.Hedgehog (testPropertyNamed)
 
--- | Proves two list are equivalant, but not identical
-
--- NOTE --
--- | UtxoEvents equivalent relationship
--- Not all utxoEvent attributes have defined `Eq` and/or `Ord` relationship defined.
--- As events are disassembled and reassembled, the Ordering of these sub-parts may change in the coresponding collections.
--- Therefore we used the Equivalence relationship to show two event are morally equal.
-equivalentLists :: Eq a => [a] -> [a] -> Bool
-equivalentLists us us' =
-  length us == length us'
-  &&
-  all (const True) [u `elem` us'| u <- us]
-  &&
-  all (const True) [u `elem` us| u <- us']
-
 tests :: TestTree
 tests = testGroup "Spec.Marconi.ChainIndex.Indexers.Utxo"
     [ testPropertyNamed
-        "filter UtxoEvent for Utxos with address in the TargetAddress"
+        "marconi-chain-index-utxo filter UtxoEvent for Utxos with address in the TargetAddress"
         "eventsAtAddressTest"
         eventsAtAddressTest
 
     , testPropertyNamed
-        "marconi-utxo event-to-sqlRows property"
-        "eventsToRowsRoundTripTest"
+        "marconi-chain-index-utxo roundtrip event-to-sqlRows conversion property"
+        "UtxoEvents to Sql Row presentation round trip conversion"
```

Comment:
IIRC That parameter needs to be the same name at the function. So you need to change for the tests below as name. Only the first parameter of `testPropertyNamed` gives a descriptive name.